libbeat/scripts/load-dashboards.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+# Usage examples:
+# env KIBANA_INDEX='.kibana_env1' ./load.sh
+# ./load.sh -url http://test.com:9200
+# ./load.sh -url http://test.com:9200 -user admin:secret
+# ./load.sh -url http://test.com:9200 -index .kibana-test
+
+# The default value of the variable. Initialize your own variables here
+ELASTICSEARCH=http://localhost:9200
+CURL=curl
+KIBANA_INDEX=".kibana"
+KIBANA_PATH?=""
```

Comment:
That is the idea. As this is WIP it is not implemented this way yet here. As this PR is getting to big I plan to split it up in multiple parts and as a first step only move the dashboards over and then get all the new tooling in place.